src/main/java/org/minions/devfund/katerinaanzoleaga/moviedatabase/Movie.java

```diff
@@ -0,0 +1,92 @@
+package org.minions.devfund.katerinaanzoleaga.moviedatabase;
+
+
+import java.util.ArrayList;
+
+/**
+ * Movie database.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Movie constructor to be defined the way we are going to set the actors list
+     */
+    public Movie(String name) {
+        this.name = name;
+        this.rating = 0;
+        this.actors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)